src/main/resources/starter.json

```diff
@@ -47,81 +48,98 @@
       ]
     }
   ],
-  "stack": [
+  "vertxStack": [
```

Comment:
It will be simpler to maintain this file if we:

- keep a single stack definition
- add a `mutinyBindings` boolean attribute to items

The attribute value would indicate if the module has Mutiny bindings.

Reply:
done in 03b4d28.
I felt pretty dirty creating that abomination of a json document initially but I thought it would be nicer than adding the mutiny prefix in the java code later. In hindsight, I like your approach a lot more. Much cleaner 👍